ansible_collections/arista/avd/roles/eos_cli_config_gen/README_v4.0.md

```diff
@@ -786,7 +786,8 @@ hardware_counters:
 tcam_profile:
   system: < tcam profile name to activate >
   profiles:
-    < tcam_profile 01 >: "{{ lookup('file', '< path to TCAM profile using EOS syntax >') }}"
+    - name: < tcam_profile 01 >
+      file: "{{ lookup('file', '< path to TCAM profile using EOS syntax >') }}"
```

Comment:
strictly speaking this is not a file. We use a lookup module as inline jinja in the example here, but the variable contents is actually the full profile config.
I suggest `config` as the key. @carlbuchmann

Reply:
I agree let's update to `config` in this PR.

A future enhancement would be to add a new option/key called `config_file_path` option.
This would simplify user input when and avoiding inline jinja (as per shown example), and wanted to store this in an external file, as tcam profile configurations can be lengthy!

Reply:
Updated.